operations/onboarding.md

```diff
@@ -0,0 +1,111 @@
+
+## Lobbying and Direct Messages
+
+Historically, the TOC has had a problem with “lobbying”, e.g. a project approaching individual TOC members for feedback / comment, often  moving on to another TOC member if they didn’t get the answer they wanted. This was particularly true for the old Sandbox model and was a key reason for changing from sponsorship to a voting model. With this in mind, TOC members should disclose to the rest of the TOC if they have been approached by a project or individual within the community or have any hard or soft conflicts of interest. It is acceptable and expected that projects or community members will reach out to the TOC for advice, however to maintain consistency in expectations, *all TOC members should be made aware of those discussions, with the opportunity to provide additional information or recommendations*. This ensures the TOC maintains a strong, clear voice to our technical communtiy, consistent with our processes, [values](https://github.com/cncf/foundation/blob/main/charter.md#3-values), [principles](../PRINCIPLES.md), and the Charter.
+
+As the TOC continues to work through updating our documentation on the repo, older references to "seeking sponsorship" may result in projects reaching out to TOC members directly to "sponsor" their project for moving levels.  When this occurs, kindly direct them to the Moving Levels resources which details how projects are recommended for promotion to the next level.  The TOC strives to pick up projects from the backlog in the order which they are recommended for promotion, while balancing alignment of TOC member expertise, liaisoning assignment, and availability of our members. 
+
+## Public persona and engagement
+
+As a member of the TOC, you may receive requests to speak at conferences, community days or events, or even on podcasts and webcasts and contribute to or participate in panels, blogs, and articles. It is important that you are aware of the capacity that you are participating in these activities in.  Depending on the activity, particularly for conferences and community days or events facillitated by the CNCF, support may be available to allow a TOC member to attend and speak. If you are approached to do this for a CNCF facillitated event and interested, reach out to CNCF Staff.
```

Comment:
would be good to specify which CNCF staff or a mailing list?

Reply:
I think CNCF staff is good enough